PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -34,6 +34,7 @@
 )
 run2_miniAOD_80XLegacy.toModify( slimmedElectronsUpdated, computeMiniIso = True )
 # bypass the update to 106X in 106X to only pick up the IP sign fix
+run2_egamma_2016.toModify(slimmedElectronsUpdated, src = cms.InputTag("slimmedElectrons"))
 run2_egamma_2017.toModify(slimmedElectronsUpdated, src = cms.InputTag("slimmedElectrons"))
 run2_egamma_2018.toModify(slimmedElectronsUpdated, src = cms.InputTag("slimmedElectrons"))
 run2_nanoAOD_106Xv1.toModify(slimmedElectronsUpdated, src = cms.InputTag("slimmedElectrons"))
```

Comment:
line 37-40 I think you can remove:
This src = cms.InputTag("slimmedElectrons") seems the default for the whole Run2
so you can set in line 27
Then you need only customise with slimmedElectronsTo106X and you do it anyway below for EOY.

Seems a general pattern for the things below every time you have 
modifier in run2_egamma_2016,run2_egamma_2017,run2_egamma_2018